recipe-box-frontend/src/main/frontend/src/components/RecipesList/RecipesGrid.tsx

```diff
@@ -29,11 +29,18 @@ export default function RecipesGrid({
         const signal = controller.signal;
         const userId = storedUserId();
 
-        let url = `${backendUrl}/recipes/${mealType ? `?mealType=${mealType}` : ""}`;
```

Comment:
Pull from main please, I have a feeling that a recently viewed pr modified something here